src/rest.cpp

```diff
@@ -608,7 +608,20 @@ static bool rest_mempool(const std::any& context, HTTPRequest* req, const std::s
     case RESTResponseFormat::JSON: {
         std::string str_json;
         if (param == "contents") {
-            str_json = MempoolToJSON(*mempool, true).write() + "\n";
+            std::string raw_verbose = req->GetQueryParameter("verbose").value_or("true");
```

Comment:
maybe use a `const` here?
```suggestion
            const std::string raw_verbose = req->GetQueryParameter("verbose").value_or("true");
```